### ISSUE TITLE
Unable to delete tenant without MapModule

### ISSUE THREAD

**Issue by ldidry** [it-novum/openITCOCKPIT#1230]
**Describe the bug**
On a fresh installation (on Debian 10), following the procedure on https://openitcockpit.io/download_server/, I can’t delete a tenant if MapModule is not installe

**To Reproduce**
1. Install openITCOCKPIT following the instructions of https://openitcockpit.io/download_server/
2. Create a tenant
3. Try to delete a tenant
4. Error 500

**Expected behavior**
Tenant should have been deleted

**Versions**
* openITCOKPIT Server Version: 4.2.3
* Debian 10

**Additional context**
If debug is activated, `frontend/error.log` says:

```
Error: [BadMethodCallException] Unknown method "getMapsByContainerIdExact" called on Cake\ORM\Table in /opt/openitc/frontend/vendor/cakephp/cakephp/src/ORM/Table.php on line 2618
```

`getMapsByContainerIdExact` function is defined in `frontend/plugins/MapModule/src/Model/Table/MapsTable.php`. Installing the MapModule solves the problem.

Suggestion: either install MapModule while installing openITCOCKPIT or define the `getMapsByContainerIdExact` function in the core.

**Comment by m4ximum** [comment]
Hi ldidry,
this error is already known and fixed in #1206
It will be rolled out with the upcoming version.
thank you for your report.

**Comment by m4ximum** [comment]
ITC-2479
